packages/firestore/test/util/helpers.ts

```diff
@@ -129,6 +129,10 @@ export function deletedDoc(
   return new NoDocument(key(keyStr), version(ver));
 }
 
+export function removedDoc(keyStr: string): NoDocument {
```

Comment:
So deletedDoc() returns a NoDocument with a specific version and removedDoc() returns a NoDocument with a forDeletedDoc() version.  This seems kinda' strange. :-)

How about just a noDoc() function that takes an optional version or something?

Reply:
I'm inclined to leave this as is as it matches the prodo semantics:

- A documentDelete message indicates that the document was deleted. That's why `deletedDoc` takes the version of the delete.

- A documentRemove just means that the document is no longer accessible, and we synthesize a delete locally.

Reply:
Agree to disagree!  These helpers are just helpers for creating our model objects and shouldn't really have protocol semantics embedded into them...  

That said, I care about this much: --><--  And it's probably a pain to update existing usages. So whatevs.